docs/pages/versions/unversioned/config/babel.mdx

```diff
@@ -0,0 +1,162 @@
+---
+title: babel.config.js
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/babel-preset-expo'
+packageName: 'babel-preset-expo'
+---
+
+The default Babel config for Expo projects uses the preset `babel-preset-expo`:
+
+```js babel.config.js
+module.exports = function (api) {
+  api.cache(true);
+  return {
+    presets: ['babel-preset-expo'],
+  };
+};
+```
+
+## Envrionment Variables
+
+Projects using `babel-preset-expo`, have access to the following environment variables:
+
+- `process.env.EXPO_OS`: The operating system of the device. This can be used as a replacement for `Platform.OS`. One of `ios`, `android`, `web`.
```

Comment:
```suggestion
- `process.env.EXPO_OS`: The operating system of the device. This can be used as a replacement for `Platform.OS`. One of `android`, `ios`, `web`.
```

Platform order nit.